picard/ui/infodialog.py

```diff
@@ -206,18 +225,17 @@ def _display_artwork_type(self):
     def arrange_images(self):
         def get_image_type(image):
             return image.types_as_string()
-        self.obj.metadata.images.sort(key=get_image_type)
-        if self.display_existing_artwork:
-            self.obj.orig_metadata.images.sort(key=get_image_type)
+        self.images.sort(key=get_image_type)
```

Comment:
Looking at the number of times this has been done, maybe we can make an ImageList class and implement __cmp__ along with other default initializations for the class?

Will help in reducing the ```if elses``` and make it easier to fix bugs if all of the code is compacted to one python module.